common/algorithms/simd-int-to-string-test.cpp

```diff
@@ -11,23 +11,27 @@
 
 namespace {
 std::string int32_to_string(int32_t x) {
-  char buffer[11];
-  return std::string{buffer, simd_int32_to_string(x, buffer)};
+  const size_t buffer_size = 11;
+  char buffer[buffer_size];
```

Comment:
please, use std::array<char, 11> in c++ code, or even std::array<std::byte, 11> having c++17

Reply:
same for code below

Reply:
done